### PR TITLE
Tests fixes

### PR DESCRIPTION
**Pull request by kaimast**
Let me know if this breaks anything for you.

### CODE REVIEW

File: include/transform.h [hunk 1]
```diff
@@ -4862,9 +4860,9 @@ namespace functions {
                                     resultStridesIter);
 
                             //pointer to where max value would be
-                            if(shape::order(resultShapeBuffer) == 'c' || shape::order(resultShapeBuffer) == 'f' &&
+                            if(shape::order(resultShapeBuffer) == 'c' || (shape::order(resultShapeBuffer) == 'f' &&
                                                                          maxIdx * shape::stride(resultShapeBuffer)[shape::rank(resultShapeBuffer) - 1] >=
-                                                                         shape::length(resultShapeBuffer))
```
Comment (kaimast) [inline include/transform.h]: I am not 100% sure if this change is correct. But GCC complains if you don't set a parenthesis here.

**Comment by agibsonccc** [comment]
I can't do const. I got rid of that for a reason.

**Comment by kaimast** [comment]
So which of these functions are actually invoked by JNI? only the ones in include/*?

**Comment by agibsonccc** [comment]
Basically, anything in the blas folder

**Comment by kaimast** [comment]
Oh, but none of these changes are in the blas folder unless im mistaken?

**Comment by agibsonccc** [comment]
They get run under the covers.

**Comment by agibsonccc** [comment]
Cool thanks!

**Pull request merged**